### DISCUSSION TITLE
Filter by "no version" OR "version:0.2.0"

### DISCUSSION THREAD

**Discussion by TreyWW** [community/community#78067] [Projects and Issues] [answered]
### Select Topic Area

Question

### Feature Area

Projects

### Body

Hi there,

Today I was trying to use a filter for my  board, i want it to have:
- Any items without any version attached
OR 
- Any items with the label "0.2.0"
BUT NOT ANY OTHER ITEMS THAT HAVE A VERSION THAT != 0.2.0 or NONE

I could't figure this out. 

I need this pretty much `version:0.2.0` OR `no:version`

Any way I could do this?
Thanks!

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by treeder** [comment, marked as answer]
Would be very nice. Asked 2 years ago here: https://github.com/orgs/community/discussions/16456

**Reply by TreyWW** [reply]
Ah okay, apologies didn't see that discussion.
